src/toil/batchSystems/parasol.py

```diff
@@ -133,28 +104,45 @@ def __init__(self, config, maxCores, maxMemory, maxDisk):
         #by themselves are removed in getUpdatedJob, and jobs
         #that are killed are removed in killBatchJobs.
         self.runningJobs = set()
-         
+
     def issueBatchJob(self, command, memory, cores, disk):
         """Issues parasol with job commands.
         """
         self.checkResourceRequest(memory, cores, disk)
-        pattern = re.compile("your job ([0-9]+).*")
-        parasolCommand = "%s -verbose -ram=%i -cpu=%i -results=%s add job '%s'" % (self.parasolCommand, memory, cores, self.parasolResultsFile, command)
 
+        megabyte = 1e6
+        roundedMemory = int(memory/megabyte) * megabyte
```

Comment:
Rounding down is often referred to as truncation. The name of the variable should reflect that. Second question is if we want to truncate to MB or MiB, i,e, 10^6 or 2^20. Third, when I see floating point math in a situation were discrete values are desired, alarm bells go off inside my head. Floating point numbers are discrete, too, of course but the rules that govern truncation and rounding are more complicated than with integer math. You might just want to do

```
MiB = 1 << 20 # or 1024 * 1024
truncatedMemory = memory / MiB * MiB
```
In the old days, people did this

```
truncatedMemory = memory & -1 << 20
```
Depending on how familiar people are with bit manipulation, twos-complement and so on, people would prefer one or the other. I do like to stay in the discrete integer domain.